### PR TITLE
fix: Fix CCF extended api call

### PR DESCRIPTION
**Pull request by Nurkambay**
## How Has This Been Tested?

This fix can be tested on Windows from the CCF side

## Checklist:
- [x] Each of my commits contains one meaningful change
- [x] I have performed a rebase of my branch on top of the development
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by ishche — APPROVED** [review]

**Review by amanPras — APPROVED** [review]
👍 Tested !

File: clients/cobol-lsp-vscode-extension/src/__tests__/services/LanguageClientServiceTest.spec.ts [hunk 1]
```diff
@@ -37,13 +37,19 @@ jest.mock("vscode", () => ({
   window: {
     createOutputChannel: jest.fn(),
   },
+  Uri: {
```
Comment (amanPras) [inline clients/cobol-lsp-vscode-extension/src/__tests__/services/LanguageClientServiceTest.spec.ts:40]: Probably we can revert this change
